src/style/mixins.scss

```diff
@@ -87,15 +75,11 @@
 
 @mixin iui-line-clamp($lines: 1) {
   overflow: hidden;
-  white-space: nowrap;
   text-overflow: ellipsis;
-
-  @supports (-webkit-line-clamp: 1) {
-    white-space: unset;
-    display: -webkit-box; /* stylelint-disable-line */
-    -webkit-line-clamp: $lines;
-    -webkit-box-orient: vertical; /* stylelint-disable-line */
-  }
+  white-space: unset;
```

Comment:
should not need to unset anymore as you removed `nowrap`

Reply:
fixed